src/kayenta/report/list/table.tsx

```diff
@@ -12,7 +15,70 @@ import { PipelineLink } from './pipelineLink';
 
 import './executionList.less';
 
+const isAtlasScope = (scope: string, metrics: ICanaryMetricConfig[]) => (
+  metrics.some(({ query, scopeName }) => scopeName === scope && query.type === 'atlas')
+);
+
+const getScopeLocations = (scopes: ICanaryScopesByName, metrics: ICanaryMetricConfig[]) => (
+  Object.keys(scopes).reduce<Set<string>>((acc, scopeName) => {
+    const { controlScope, experimentScope } = scopes[scopeName];
+    const isAtlas = isAtlasScope(scopeName, metrics);
+
+    // When atlas metrics have the dataset param set to global,
+    // the location field is not accurate.
+    if (isAtlas && get(controlScope, 'extendedScopeParams.dataset') === 'global') {
+      acc.add('Global');
+    } else {
+      acc.add(controlScope.location)
+    }
+    if (isAtlas && get(experimentScope, 'extendedScopeParams.dataset') === 'global') {
+      acc.add('Global');
+    } else {
+      acc.add(experimentScope.location)
+    }
+
+    return acc;
+  }, new Set())
+);
+
 const columns: ITableColumn<ICanaryExecutionStatusResult>[] = [
+  {
+    label: 'Score / Report',
```

Comment:
Up to you, but my first read of text having the format `A / B` is A or B, whereas I'd read `A + B` as A and B.

Reply:
I actually agree somewhat — I went back and forth on it a bit, and was seriously considering just calling it `Summary`. I don't think this header actually provides much informational value, given that the score / date are pretty foundational concepts and will likely already have enough visual cues to make sense.

Do you have 👍 / 👎  opinions on `Summary`?

Reply:
`Summary` seems fine. I agree that this particular header doesn't provide much value (and it would probably look strange to omit just this one).

Reply:
👍 Summary it is then